openreview/venue/process/invitation_edit_process.py

```diff
@@ -135,6 +135,13 @@ def post_invitation(note):
             content['replyto'] = { 'value': note.id }
             content['replytoSignatures'] = { 'value': note.signatures[0] }
 
+        if 'replyto' in invitation.edit['content'] and 'reviewNumber' in invitation.edit['content']:
+            paper_number = note.invitations[0].split(submission_name)[-1].split('/')[0]
+            content['noteId'] = { 'value': note.forum }
+            content['noteNumber'] = { 'value': int(paper_number) }
+            content['replyto'] = { 'value': note.id }
+            content['reviewNumber'] = { 'value': note.number }
```

Comment:
I think we should use a more general name here, like `replyNumber` because we can create invitations are replies of reviews, metareviews, etc

Reply:
yes! I realized this when I started to update the CustomStage